src/main/java/InvalidTaskNumber.java

```diff
@@ -0,0 +1,8 @@
+public class InvalidTaskNumber extends Exception{
```

Comment:
I like how you have handled various possible errors with different exceptions. Keep up the good work!